samples/excel/85-preview-apis/range-merged-areas.yaml

```
@@ -0,0 +1,120 @@
+order: 7
+id: excel-merged-areas
+name: Merged areas
+description: This sample shows how to create and then find the merged areas in a worksheet.
```

Suggestions for brevity.
```suggestion
description: This sample shows how to create and find merged areas in a worksheet.
```